src/dataMigration.js

```diff
@@ -73,6 +73,27 @@ const dataMigrations = [
           }
         });
       });
+
+      const nameResults = database.objects('Name').filtered('id = $0',
+                                settings.get(SETTINGS_KEYS.SUPPLYING_STORE_NAME_ID));
+      if (nameResults.length < 1) throw new Error('Supplying Store Name ID missing from settings');
+      const mainSupplyingStoreName = nameResults[0];
+
+      const requisitions = database.objects('Requisition')
+                                   .filtered('otherStoreName = Null').snapshot();
+
+      database.write(() => {
+        requisitions.forEach(requisition => {
+          if (!requisition.isRequest) return;
```

Comment:
I think it should be fine to add a otherStoreName as mainSupplyingStoreName for any requisition prior to v3. As they will only be requisitions for main supplying store